crt_portal/static/sass/custom/landing.scss

```diff
@@ -18,6 +18,18 @@ $arrow-height: 24px;
   li {
     margin-bottom: 1rem;
   }
+
```

Comment:
👆 On line 13, the chunk of code that looks like this:
```
h2 {
  margin-top: 1.5rem;
  margin-bottom: 0;
}
```

should be changed to:

```
h2 {
  margin-top: 0;
  margin-bottom: 0;
  @include at-media(tablet) {
    margin-top: 1.5rem;
  }
}
```

This'll automatically adjust the card margins correctly (once you also delete the rule for `.crt-landing--already_submitted h2` down below; I left a note).